packages/explat-react-client/src/test/use-experiment.ts

```diff
@@ -0,0 +1,52 @@
+import type { ExPlatClient, ExperimentAssignment } from '@automattic/explat-client'
+import {renderHook, act} from '@testing-library/react-hooks'
+import { validExperimentAssignment } from '@automattic/explat-client/src/internal/test-common'
+
+import createExPlatReactClient from '../index'
+
+
+const createMockExPlatClient = (isDevelopmentMode: boolean = false): ExPlatClient => ({
+    loadExperimentAssignment: jest.fn(),
+    dangerouslyGetExperimentAssignment: jest.fn(),
+    config: {
+        isDevelopmentMode,
+        logError: jest.fn(),
+        fetchExperimentAssignment: jest.fn(),
+        getAnonId: jest.fn(),
+    }
+})
+
+const createControllablePromise = <T>() => {
+    let resOuter, rejOuter
+
+    const promise = new Promise<T>((resolve, reject) => {
+        resOuter = resolve
+        rejOuter = reject
+    })
+
+    return {
+        resolve: (resOuter as unknown) as (T) => void,
+        reject: (rejOuter as unknown) as (...x: unknown[]) => void,
+        promise,
+    }
+}
+
+describe('useExperiment', () => {
+    it('should correctly load an experiment assignment', async () => {
+        const ExPlatClient = createMockExPlatClient();
```

Comment:
The variable name should start with a lowercase character and not shadow the type.

Reply:
Huh, weird how these aren't being picked up by husky.